requirements-dev.txt

```diff
@@ -5,3 +5,4 @@ flake8==3.7.9
 pylint==2.4.4
 pytest==5.4.1
 requests==2.23.0
+yapf==0.29.0
```

Comment:
Could the yapf version be changed to `0.27.0`- That is the version we are running on the Fedora side of things so the output would match.

Reply:
sure thing